src/bindings/c/tests/ov_c_api_test.cpp

```diff
@@ -2,17 +2,18 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#include "c_api/ov_c_api.h"
```

Comment:
Let's split this file to the set of small separate files

Reply:
done